SQL/beestation_schema.sql

```diff
@@ -568,6 +568,14 @@ CREATE TABLE `achievement_metadata` (
 	PRIMARY KEY (`achievement_key`)
 ) ENGINE=InnoDB;
 
+DROP TABLE IF EXISTS `metacoin_item_purchases`;
+CREATE TABLE IF NOT EXISTS `metacoin_item_purchases` (
```

Comment:
this table cannot exist because you are dropping it one line above

Reply:
@Crossedfall

Reply:
Firstly, I don't know why you're @'ing me. I didn't write that nor is this my PR. Secondly, you're wrong.

The process is: "If the table exists, delete it. If the table doesn't exist, create it." Not the other way around.

Reply:
```suggestion
CREATE TABLE IF NOT EXISTS `SS13_metacoin_item_purchases` (
```

I need to split this into two suggestions because GitHub is having a moment.